src/lofreq/snpcaller.c

```diff
@@ -1100,6 +1102,14 @@ snpcaller(long double *snp_pvalues,
         goto free_and_exit;
     }
 
+    double mu = 0;
+    for (int i = 0; i < num_err_probs; ++i) {
+        mu += err_probs[i];
+    }
+    const long double poibin_approximation = 1 - gsl_cdf_poisson_P(max_noncons_count - 1, mu);
+    if (poibin_approximation > sig_level + 0.01) {
```

Comment:
The cutoff of `0.01` is rather loose and should probably also be a function of `n`, as the error decreases with increasing sample count.